### PR TITLE
feat: partial neurons and canisters card list new design

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Partial implementation of the new design for the "Neurons" card list with some tweak for UX reason discussed with Mischa.

# Changes

- `.card-list`
- "merge neuron" becomes secondary action
- extract `<NeurontTitle />` component from existing
- present information as in design with few ux tweak
- since I was already there, also reviewed briefly the canisters list even though there is no design yet
- fix tooltip positioning for new design (again)

# Screenshots

Neurons:

Capture d’écran 2022-11-18 à 12 11 05
Capture d’écran 2022-11-18 à 12 11 08
Capture d’écran 2022-11-18 à 12 11 13
Capture d’écran 2022-11-18 à 12 11 20

Canisters:

Capture d’écran 2022-11-18 à 13 39 26
Capture d’écran 2022-11-18 à 13 39 29
Capture d’écran 2022-11-18 à 13 39 31
Capture d’écran 2022-11-18 à 13 39 35

### CODE REVIEW

File: frontend/src/lib/components/sns-neurons/SnsNeuronCard.svelte [hunk 1]
```diff
@@ -57,21 +57,23 @@
...
 
-  <div slot="end" class="currency">
-    {#if neuronStake !== undefined}
-      <AmountDisplay amount={neuronStake} />
-    {:else}
-      <Spinner inline size="small" />
-    {/if}
-  </div>
+  <div class="content">
+    <div class="currency">
```
Comment (mstrasinskis) [inline frontend/src/lib/components/sns-neurons/SnsNeuronCard.svelte:67]: Is the class `currency` in use here?

File: frontend/src/lib/components/canisters/CanisterCardTitle.svelte [hunk 2]
```diff
@@ -13,29 +13,24 @@
...
 </div>
 
 <style lang="scss">
-  .h3 {
-    :global(button) {
-      margin-bottom: var(--padding-0_5x);
-    }
-  }
-
-  .h1 {
```
Comment (mstrasinskis) [inline frontend/src/lib/components/canisters/CanisterCardTitle.svelte:31]: Does `h1` need extra styling?

**Review by mstrasinskis — APPROVED** [review]
LGTM, just a couple minor questions?

File: frontend/src/lib/components/sns-neurons/SnsNeuronCard.svelte (hunk 1 above)
Comment (peterpeterparker) [reply frontend/src/lib/components/sns-neurons/SnsNeuronCard.svelte:67]: Not anymore, good catch 👍

File: frontend/src/lib/components/canisters/CanisterCardTitle.svelte (hunk 2 above)
Comment (peterpeterparker) [reply frontend/src/lib/components/canisters/CanisterCardTitle.svelte:31]: Not anymore at least, looks ok without overriding the global that set margin bottom to the button so I removed this style.

Capture d’écran 2022-11-19 à 08 12 12

**Pull request merged**